### DISCUSSION TITLE
how do i add the district number in the meeting form?

### DISCUSSION THREAD

**Discussion by dennisguilbault** [code4recovery/12-step-meeting-list#577] [Q&A] [answered]
I need to add the district number for a meeting, but can't find where to do this.  Can you instruct me please?

**Comment by brianw-area24** [comment]
In the 12 Step Meeting List (TSML) plugin, the district number is only available for Groups, not for individual meetings.

**Comment by Grateful-Viv** [comment, marked as answer]
You add your district numbers under Meetings > Districts 

When you go to edit/add a meeting that is part of a group, toggle on the field **part of a group**, then enter Group Name and select the District number from the dropdown.
